2-api/key-concepts.md

```diff
@@ -43,23 +49,24 @@ rest of the comment.
 
 ### Span scores
 
-Model scores for particular subparts of the request's comment. For example, if the API only found one sentence in a paragraph to be "toxic", it could return a high "toxic" span score for the span corresponding to that sentence while giving a low "toxic" span score to the rest of the comment. This score is only available for some models.
+A **span score** is a model score for subparts of the request's comment.
 
-## Context
+For example, if the API only found one sentence in a paragraph to be "toxic",
+it could return a high "toxic" span score for the span corresponding to that
+sentence while giving a low "toxic" span score to the rest of the comment.
+This score is only available for some models.
 
-*(Coming soon)* **Context** is a representation of the conversation context
-for the comment; for example, an article that is being commented on, or
-another comment that is being replied to. It is used in the analysis of the
-comment text; for example, when determining the "off-topic" model score.
+### Summary Score
 
-**Note**: you can send context with your requests, however our current models
-are not making use of it&mdash;so sending context won't change any model's
-scores.
+A model's **summary score** provides an overall score for the entire comment.
+While the API may return multiple span scores for a given input, it will
+always return exactly one summary score.
 
 ## Toxicity
 
-One of the attributes that Perspective can score. Toxicity is defined as "a rude,
-disrespectful, or unreasonable comment that is likely to make you leave a discussion."
+**Toxicity** is an attribute which Perspective can score. Toxicity is defined
```

Comment:
Perhaps capitalize TOXICITY here. OK to bold it, too, if you like.

Reply:
👍